### ISSUE TITLE
Synchronize two different kdbx files

### ISSUE THREAD

**Issue by clustor** [keeweb/keeweb#1917]
I have one kdbx file on the disk and another synced to Google Drive. I am trying to get the changes in the file on the disk synced to the file on Google Drive. 

I can open the file on local disk and try to save to Google Drive but it says file already exists without offering me a choice to synchronize. I don't want to overwrite without knowing it will sync the changes.  Another option is to export and import but I would like to make sure the entries will be synchronized. 

I have a feeling this can be done but it would be great if the steps can be provided.

**Comment by clustor** [comment]
I used to do that KeePass in the past before I switched to KeeWeb. Wanted to check if this can be done without me trying to find a Windows PC. KeePass with mono on Mac had its own issues.

**Comment by antelle** [comment]
You can replace a file in filesystem and then KeeWeb will sync it, but it's rather hard to understand how you need to do it, so it's safer to say that manual merge doesn't exist in KeeWeb.

**Comment by clustor** [comment]
Yes. This isn't straightforward without having explicit "Sync from another database" dialogue. If I have a kdbx on Google Drive and if I replace the file on Google Drive with a new one using upload, does Keeweb sync the changes when it opens or saves the file next time and realizes it's different from cached copy? 

FWIW, I used KeepassXC available for Mac to merge my files and I am back to using KeeWeb.
